open-craft/configuration
# SE-1617, SE-2298 Makes the ecommerce OSCAR_DEFAULT_CURRENCY configurable.

Allows ecommerce deployments to configure their default currency, using the `ECOMMERCE_OSCAR_DEFAULT_CURRENCY` variable.  [Default is set to `"USD"` in the ecommerce platform](https://github.com/edx/ecommerce/blob/4b48dc739b42efdd746ede627bffdc7c4ad3d438/ecommerce/settings/_oscar.py#L99).

Upstream PR TBD.

**Dependencies**: https://github.com/open-craft/ecommerce/pull/13

**Sandbox URL**: Tested on client instance.

**Testing instructions**:

1. Set `ECOMMERCE_OSCAR_DEFAULT_CURRENCY` and deploy a new appserver using this branch and the ecommerce branch from https://github.com/open-craft/ecommerce/pull/13.
1. Ensure that the ecommerce `settings.OSCAR_DEFAULT_CURRENCY` matches the setting.

**Reviewers**
